spec/presenters/history_list_spec.rb

```diff
@@ -0,0 +1,20 @@
+describe HistoryList do
+  describe 'contents' do
+    it "starts with proposal creation" do
+      proposal = create(:proposal)
+
+      history = HistoryList.new(proposal)
+      expect(history.events.first.event).to eq("create")
+    end
+
+    describe "filtering" do
```

Comment:
perhaps use `context ` here rather than `describe` since we're still testing `#events`

```ruby
context "events include work order updates" do
  it "excludes client data events" do
    ...
  end
end
```